test/test_data/README (1).txt

```diff
@@ -0,0 +1 @@
+This is an example readme file
```

Comment:
I think both this file and the README in this directory should be renamed to something like "mock_study_doc_upload.txt" and "mock_study_doc_upload2.txt", so that they are not confused with READMEs for the code.  

Also, does running this test delete the file?  if so, the test should copy the file to a temp directory, and upload that, so that it can be run without modifying files under source control, and run successfully more than once. See, e.g. lines 30-33 of SyntheticStudyPopulator.rb

Reply:
Good points.

Reply:
Actually, for the 2nd point - we patched the uploader for study files so that in the test environment, it performs a copy rather than a move.  See [here](https://github.com/broadinstitute/single_cell_portal_core/blob/development/app/uploaders/upload_uploader.rb#L29) and [here](https://github.com/broadinstitute/single_cell_portal_core/blob/development/app/uploaders/upload_uploader.rb#L33).  So we shouldn't have to worry about that in tests.

Reply:
ah, thanks for reminding me, good point